regression/contracts/assigns_enforce_free_dead/test.desc

```diff
@@ -40,4 +42,4 @@ We also check (using positive regex matches above) that
 `**p` should be assignable in one case and should not be assignable in the other.
 
 Finally, we check that there should be no "internal" assertion failures
-on `tmp_cc` variables used to track CARs.
+on `__car_valid`, `__car_ub`, `__car_lb` variables used to track CARs.
```

Comment:
Changes in this file should be folded into the commit introducing the new names.